wp-content/themes/eighshift-task/src/blocks/custom/posts/assets/load-more.js

```diff
@@ -0,0 +1,61 @@
+import '@babel/polyfill';
+
+export class LoadMore {
+
+  constructor(element) {
+    this.element = element;
```

Comment:
could you put "options" object in constructor and then use that here like
this.element = option.element

this helps you to have more robust class